packages/@react-aria/interactions/src/useLongPress.ts

```diff
@@ -81,6 +81,12 @@ export function useLongPress(props: LongPressProps): LongPressResult {
         timeRef.current = setTimeout(() => {
           // Prevent other usePress handlers from also handling this event.
           e.target.dispatchEvent(new PointerEvent('pointercancel', {bubbles: true}));
+
+          // Ensure target is focused. On touch devices, browsers typically focus on pointer up.
+          if (getOwnerDocument(e.target).activeElement !== e.target) {
+            focusWithoutScrolling(e.target as FocusableElement);
+          }
```

Comment:
Perhaps not a blocker, but just noticed that long pressing on "Two" in ["TableView: static" story w/ highlight selection and multiple selection](https://reactspectrum.blob.core.windows.net/reactspectrum/f7e4d9e220b5afbeecd1be5498070e39a77a6db5/storybook/index.html?path=/story/tableview--static&args=selectionMode:multiple;selectionStyle:highlight&providerSwitcher-express=false) via Chrome Android causes a focus ring to appear on the "One" cell instead of "Two". Doesn't seem to happen on my iPad though